api/MediaStreamTrackAudioSourceNode.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaStreamTrackAudioSourceNode",
         "support": {
           "chrome": {
-            "version_added": null
+            "version_added": false
```

Comment:
This looked a bit suspect, but looks like it's the return value of https://webaudio.github.io/web-audio-api/#dom-audiocontext-createmediastreamtracksource which isn't supported.